### ISSUE TITLE
Fix case sensitivity with services dropdown

### ISSUE THREAD

**Issue by awalker4** [studentinsights/studentinsights#581]
Educator names are currently case sensitive in the dropdown for services.

![case sensitive](https://cloud.githubusercontent.com/assets/3846783/16100866/5db0df0e-332f-11e6-9317-b674b24e7c91.gif)

Ideally typing either "Teacher" or "teacher" would give us the right option. If we type "teacher", we should be able to jump straight to the correct entry without having to do anything.

**Comment by awalker4** [comment]
I'll give anyone else a chance at this before I start looking into it.

**Comment by kevinrobinson** [comment]
@awalker4 great catch! 😄

**Comment by Jrafalak** [comment]
I'm wondering if we do want to hide a name with different capitalization patterns after all... Off the top of my head, what if there were two unique teachers named "Steve Mcconnell" and "Steve McConnell"? This is definitely an edge case, but it may be safer to leave the autocomplete as is since we do allow ad hoc input. Does anyone have any thoughts on this?

**Comment by awalker4** [comment]
Hmm, definitely seems like an edgy edge case, but it could happen. I guess another argument is that most people will select the right completion long before they've typed out the whole thing. Now that I think about it, the only real "unexpected" behavior here is if you've already put in the effort to type out "teacher, sarah". I think this may be fine as is.

**Comment by kevinrobinson** [comment]
Yeah, my vote would be "the autocomplete matches in a case-insensitive way" but that the list itself shows all items without paying attention to string case.

**Comment by Jngai** [comment]
The jquery autocomplete is already case sensitive. It is working fine. If it isn't then when you type in "teacher" then "Teacher Sarah" as an option won't show up. The jumping thing occur because we have "autofocus" turned on. Here is a video with autofocus turned off. My vote is to actually turn it off so the user can explicitly choose. 

![autofocusturnedoff](https://user-images.githubusercontent.com/1547934/29526620-420fe07c-8664-11e7-9031-05a91cf05a64.gif)

"teacher, sarah" and "Teacher, Sarah" are actually two different things. You can write a hack to force the first option to be picked but I am not sure if that is a good idea.

**Comment by Jngai** [comment]
cc @alexsoble
